### PR TITLE
Implement footer links and revised copyright

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Ref https://github.com/DemocracyClub/design-system/issues/140

Of the suggestions in the original issue, these are the changes implemented (so far) in this PR: 

- [x] Split footer into two parts: links at the top (#403F41), mission statement and company info below (#000000). 
This raised questions about what to do in dark mode so for simplicity, I've used a blue line to divide the sections as shown below. 
- [x] Links at the top grouped into three categories: 'Who Can I Vote For?', 'Made by Democracy Club', 'Connect with us' (you can rename these). Each category has links related to either WCIVF, Democracy Club or social media.
- [x] Links on phone screens will use the one column layout and be displayed under each other.
- [x] Links on larger screens (from the size of a tablet screen in portrait view) will use a three column layout and be displayed in parallel columns.
- [x] Mission statement, copyright, company information at the bottom of the page 
- [x] the above, on a darker background.
- [x] Headings in white, in bolder font.
- [x] Headings, links, mission statement, copyright can use the same font sizes as it is currently or copyright could be about 2px smaller (relative to the screen size).

**Questions for Reviewers:**
- [x] What is the ideal mobile layout given that the mockup in the issue has fewer footer links that we would like have, at least on Website? 

Screenshot 2024-10-23 at 12 30 37 PM
Screenshot 2024-10-23 at 12 30 29 PM

### CODE REVIEW

**Comment by symroe** [comment]
Changed this a little:

image

File: src-site/components/footer.md [hunk 1]
```diff
@@ -6,72 +6,164 @@ Use the Footer component to add quick links, company, and contact information to
...
+          <h2>ProjectName</h2>
+          <ul>
+            <li>
+              <a href="/">Home</a>
+            </li>
+            <li>
+              <a aria-current="true" href="">About</a>
+            </li>
+            <li>
+              <a href="">Our work</a>
```
Comment (chris48s) [inline src-site/components/footer.md:110]: here

File: src-site/components/footer.md [hunk 2]
```diff
@@ -6,72 +6,164 @@ Use the Footer component to add quick links, company, and contact information to
...
+          <h2>About</h2>
+          <ul>
+            <li>
+              <a href="/">Home</a>
+            </li>
+            <li>
+              <a aria-current="true" href="/path/to/about">About</a>
+            </li>
+            <li>
+              <a href="/path/to/">Our work</a>
```
Comment (chris48s) [inline src-site/components/footer.md:133]: here

File: src-site/layout_content.md [hunk 3]
```diff
@@ -144,39 +144,24 @@
...
-          <a href="#">Donate</a>
-        </li>
-      </ul>
+    <div class="ds-footer-links">
+      <nav>
+        <h2 class="ds-h5">Contact Democracy Club</h2>
+        <ul>
+          <li><a href="/">Home</a></li>
+          <li><a aria-current="true" href="/path/to/about">About</a></li>
+          <li><a href="/path/to/">Our work</a></li>
```
Comment (chris48s) [inline src-site/layout_content.md:153]: ..and here

**Review by chris48s — COMMENTED** [review]
haven't checked it out, but the spacing in the latest screenshot looks better :+1: There's still some more links to clean up in the docs site..

**Comment by symroe** [comment]
Sorry, should be sorted now

**Review by chris48s — APPROVED** [review]

**Pull request merged**